client/components/catalog/Card/index.vue

```diff
@@ -105,19 +116,22 @@ export default {
         name,
         params: { repositoryId: this.repository.id }
       });
+    },
+    detectTruncation() {
+      const { clientWidth, scrollWidth } = this.$refs.schemaName;
```

Comment:
@mcapeta 
Would it make sense to hook into https://vuetifyjs.com/en/features/breakpoints/#breakpoint-service 🤡